src/parsers/transactions.js

```diff
@@ -286,7 +289,10 @@ const parseTransaction = (
 
     return {
       ...transaction,
-      address: toChecksumAddress(updatedAsset.address),
+      address:
+        updatedAsset.address === 'eth'
```

Comment:
should we toLower the updatedAsset.address to make it less fragile if for some reason our data provider starts sending across ETH?